ynput/OpenPype
# Houdini: Use default_variant to Houdini Node TAB Creator

## Changelog Description

Use the default variant of the creator plugins on the interactive creator from the TAB node search instead of hard-coding it to `Main`.

## Testing notes:
1. Change the default_variants for one of the Houdini creators
2. Use the Houdini node tab creation (i.e., `Create Arnold ROP`)
3. Check that the default variant on the dialog isn't `Main`
